### ISSUE TITLE
Add list of actions into the documentation file

### ISSUE THREAD

**Issue by alexanderbock** [OpenSpace/OpenSpace#2945]
Right now it is not possible for users to know what actions exist in the current profile.  It would be possible to get a list through `openspace.printInfo(openspace.action.actions())` but it is hard to parse.

It would be easier to read if we added a list of the actions into the generated documentation file

**Comment by ylvaselling** [comment]
Duplicate of #2645
